### PR TITLE
2.4.0 release

### PR DESCRIPTION
**Pull request by jsvd**
once this is merged it is necessary to cherry-pick the commit to the https://github.com/elastic/logstash/tree/special_2.4_all_packages branch where the rpm/deb are created from

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## 2.4.0 (Aug 30, 2016)
+## 2.4.0 (Aug 31, 2016)
```
Comment (ph) [inline CHANGELOG.md]: Lets add a note in the beats input.


```
This plugin only supports private key in the PKCS8 format, you can use OpenSSL to convert them see https://www.openssl.org/docs/manmaster/apps/pkcs8.html for more details.

**Comment by ph** [comment]
Added a minor notes for the beats input.

File: CHANGELOG.md (hunk 1 above)
Comment (jsvd) [inline CHANGELOG.md]: done 👍

**Comment by ph** [comment]
LGTM

**Pull request closed without merging**